sw/device/lib/dif/dif_entropy.c

```diff
@@ -93,12 +93,40 @@ dif_entropy_result_t dif_entropy_configure(const dif_entropy_t *entropy,
   return kDifEntropyOk;
 }
 
+inline static bool get_es_valid(const dif_entropy_t *entropy) {
```

Comment:
I am not sure if the `inline` is required anymore. I think the compiler does at good job at deciding when to inline.

Reply:
removed.